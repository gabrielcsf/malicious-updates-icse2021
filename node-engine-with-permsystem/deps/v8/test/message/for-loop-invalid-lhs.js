// Copyright 2015 the V8 project authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.
//
// TODO(adamk): Remove flag after the test runner tests all message tests with
// the preparser: https://code.google.com/p/v8/issues/detail?id=4372
// Flags: --min-preparse-length=0

function f() { for ("unassignable" in {}); }
