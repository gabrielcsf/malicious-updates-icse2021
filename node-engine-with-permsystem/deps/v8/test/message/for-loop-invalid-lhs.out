*%(basename)s:9: SyntaxError: Invalid left-hand side in for-loop
function f() { for ("unassignable" in {}); }
                    ^^^^^^^^^^^^^^
SyntaxError: Invalid left-hand side in for-loop

